solution/app/build.gradle

```diff
@@ -48,4 +48,13 @@ dependencies {
     implementation 'com.google.android.libraries.places:places:2.6.0'
     // Places SDK for Android KTX Library
     implementation 'com.google.maps.android:places-ktx:2.0.0'
+    implementation 'com.google.android.gms:play-services-maps:18.2.0'
+}
+
+secrets {
+    // To add your Maps API key to this project:
+    // 1. Open the root project's local.properties file
```

Comment:
Let's be consistent with the recently updated [documentation instructions](https://developers.google.com/maps/documentation/android-sdk/config#step_3_add_your_api_key_to_the_project) and add a line matching [the snippet](https://github.com/googlemaps-samples/android-samples/blob/main/snippets/app/build.gradle#L77) to specify secrets.properties